packages/cli/src/commands/init.ts

```diff
@@ -596,9 +656,14 @@ async function processInitForm(
         isSubstreams ||
         (!protocolInstance.hasContract() && !isComposedSubgraph),
       initial: initContract,
```

Comment:
If you specify `--from-source-subgraph Qmxxxxxx`, this field should have initial value `Qmxxxxxx`